spec/spec_helper.rb

```diff
@@ -57,6 +57,7 @@
     add_group 'OpenidAuth', 'modules/openid_auth/'
     add_filter 'version.rb'
     add_filter 'modules/va_facilities/lib/va_facilities/engine.rb'
+    add_filter 'app/models/nearby_facility.rb'
```

Comment:
For my knowledge: what is the purpose of adding this to `spec_helper`?

Reply:
This is what stopped the build from failing on the CI step due to not having at least 90% test coverage on `nearby_facility.rb`. Once we add the tests for the file, this can be removed.